shared/search/render.desktop.js

```diff
@@ -39,7 +39,7 @@ class Render extends Component<void, Props, void> {
         <Box style={{...globalStyles.flexBoxColumn, flex: 1}}>
           <ClearSearch {...this.props} />
           <SearchBar {...searchBarProps} />
-          <Box style={{overflowY: 'auto', height: 'calc(100% - 96px)'}}>
```

Comment:
we don't need this with flex-shrink in the other divs, also had a side effect of redrawing the box when you got tooltips on the hover